cdap-explore/src/test/java/co/cask/cdap/explore/service/BaseHiveExploreServiceTest.java

```diff
@@ -367,6 +373,16 @@ protected void configure() {
                     .build(StoreFactory.class)
           );
           bind(StreamHttpService.class).in(Scopes.SINGLETON);
+
+          install(Modules.override(new DataFabricModules().getInMemoryModules()).with(new AbstractModule() {
```

Comment:
I changed the way transaction was asserted in `HiveExploreServiceStopTestRun`, so that we don't have conditional logic in `BaseHiveExploreServiceTest` code. Now all services get stopped, and transaction state is asserted using transaction snapshot.